fabric-data-generation-api-v1/src/main/java/net/fabricmc/fabric/api/datagen/v1/provider/FabricLootTableProvider.java

```diff
@@ -52,19 +56,39 @@
 
 	FabricDataGenerator getFabricDataGenerator();
 
+	/**
+	 * Return a new exporter that applies the specified conditions to any loot table it receives.
+	 */
+	default BiConsumer<Identifier, LootTable.Builder> withConditions(BiConsumer<Identifier, LootTable.Builder> exporter, ConditionJsonProvider... conditions) {
+		Preconditions.checkArgument(conditions.length > 0);
```

Comment:
Should this have an exception message? (Also applies to all the corresponding checks in other methods)